bin/node-template/node/src/command.rs

```diff
@@ -68,8 +69,7 @@ pub fn run() -> sc_cli::Result<()> {
 		Some(subcommand) => {
 			let runner = cli.create_runner(subcommand)?;
 			runner.run_subcommand(subcommand, |config| {
-				let (builder, _, _) = new_full_start!(config);
-				Ok(builder.to_chain_ops_parts())
+				crate::service::Builder::build_ops(config)
 			})
 		}
 		None => {
```

Comment:
And then here you can call Builder::new_light and Builder::new_full instead of having functions, right?